save-orchestrator/src/main/kotlin/com/saveourtool/save/orchestrator/controller/AgentsController.kt

```diff
@@ -94,12 +89,7 @@ class AgentsController(
                     prefix = "save-execution-${execution.id}"
                 )
             }
-                .flatMap { resourcesForExecution ->
-                    val resourcesPath = resourcesForExecution.resolve(TEST_SUITES_DIR_NAME)
```

Comment:
maybe move `TEST_SUITES_DIR_NAME` to save-cloud-common and reuse in agent?

Reply:
I think it'd be even better to move this directory into agent's configuration and set from orchestrator via `agent.properties`. Because they should use the same directory for downloading tests to and to referring in execCmd to.